libmamba/src/core/util_string.cpp

```diff
@@ -7,15 +7,117 @@
 #include "mamba/core/util_string.hpp"
 
 #include <cctype>
+#include <cwchar>
 #include <cwctype>
-#include <exception>
-#include <type_traits>
+#include <stdexcept>
+
+#include <stddef.h>
 
 namespace mamba
 {
-    /***************************************************
-     *  Implementation of to_lower to_upper functions  *
-     ***************************************************/
+    /****************************************
+     *  Implementation of cctype functions  *
+     ****************************************/
+
+    char is_control(char c)
```

Comment:
Aren't all these functions supposed to return a boolean instead?

Reply:
yes, good catch !